packages/blade/src/components/Box/LayoutPrimitivesDocs.tsx

```diff
@@ -0,0 +1,462 @@
+        </ListItem>
+
+        <ListItem>
+          <ScrollIntoViewLink href="#questions-you-might-have">
+            Questions You Might Have
+          </ScrollIntoViewLink>
+          <List>
+            <ListItem>
+              <ScrollIntoViewLink href="#why-is-xyz-prop-not-support">
+                Why is xyz prop not supported?
+              </ScrollIntoViewLink>
+            </ListItem>
+            <ListItem>
+              <ScrollIntoViewLink href="#what-to-do-if-prop-is-not-supported">
+                What to do if prop is not supported?
+              </ScrollIntoViewLink>
+            </ListItem>
+          </List>
+        </ListItem>
+
+        <ListItem>
+          <ListItemLink href="/?path=/docs/components-layout-primitives-box-box--default&globals=measureEnabled:false#properties-ref">
+            Props Reference
+          </ListItemLink>
+        </ListItem>
+      </List>
+      <hr />
+      <Section id="playground">
+        <Title size="small">Playground</Title>
+        <Sandbox padding="spacing.0">
+          {`
+            import { Box } from '@razorpay/blade/components'
+
+            function App(): JSX.Element {
+              return (
+                <Box 
+                  display="flex"
+                  flexDirection={{ base: 'column', m: 'row' }}
+                  padding={{ base: ['spacing.1', '9px'], m: 'spacing.3' }}
+                >
+                  <Box 
+                    backgroundColor="surface.background.level3.highContrast" 
+                    flex="1" 
+                    minHeight="spacing.10" 
+                    minWidth="spacing.10" 
+                  />
+                  <Box 
+                    backgroundColor="surface.background.level2.highContrast" 
+                    flex="1" 
+                    minHeight="spacing.10" 
+                    minWidth="spacing.10" 
+                  />
+                </Box>
+              )
+            }
+
+            export default App;
+            `}
+        </Sandbox>
+      </Section>
+      <Section id="box-usage">
+        <Title size="small" marginBottom="spacing.3">
+          📦 Box Usage
+        </Title>
+        <Text>
+          Box is a primitive Layout component which can be used for creating different responsive
+          layouts in UI. You might have used <Code>Box</Code> in other component libraries as well
+          such as{' '}
+          <Link target="_blank" href="https://chakra-ui.com/docs/components/box">
+            Chakra
+          </Link>
+          . Our Box is similar, except it&apos;s primarily focussed towards layout properties and
+          works on all platforms.
+        </Text>
+        <Text marginY="spacing.4">The simplest Box usage would look something like this-</Text>
+        <SandboxHighlighter>
+          {`
+              <Box>Hello</Box>
+              // On Web          -> <div>Hello</div> 
+              // On React Native -> <View>Hello</View>
```

Comment:
```suggestion
              // This will translate to:
              // On Web          -> <div>Hello</div> 
              // On React Native -> <View>Hello</View>
```

Reply:
done in upcoming commit